definitions/install.yaml

```diff
@@ -6732,3 +6732,23 @@ spec:
       securityContext:
         runAsNonRoot: true
       serviceAccountName: kyverno-service-account
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app: kyverno
+    app.kubernetes.io/component: kyverno
+    app.kubernetes.io/instance: kyverno
+    app.kubernetes.io/managed-by: Kustomize
+    app.kubernetes.io/name: kyverno
+    app.kubernetes.io/part-of: kyverno
+    app.kubernetes.io/version: v1.3.6
```

Comment:
Wrong version. Let's not add this for now. I'll update it when releasing 1.5.0.

Reply:
@ojhaarjun1 - is this revmoed?

Reply:
I'll remove this. I thought you wanted this to stay and update later...